locksmith/server.js

```diff
@@ -1,4 +1,48 @@
+/* eslint no-console: 0 */
+
+const net = require('net')
 const app = require('./src/app')
 
-const port =  process.env.PORT || 8080
-app.listen(port)
+const env = process.env.NODE_ENV || 'development'
+const config = require('./config/config')[env]
+
+const port = process.env.PORT || 8080
+
+/**
+ * This is a helper function to ensure that we start the test suite only when the server is up
+ * We will retry for
+ */
+const serverIsUp = (host, port, delay, maxAttempts, callback) => {
+  let attempts = 1
```

Comment:
It feels more "correct" to me to have this initialized at 0, but this definitely works.